include/fmt/core.h

```diff
@@ -1662,7 +1662,7 @@ template <typename Context> class basic_format_arg {
   \endrst
  */
 // DEPRECATED!
-FMT_EXPORT
+//FMT_EXPORT
```

Comment:
Why comment out this?

Reply:
I had commented it out when trying to diagnose the issue which was that there was an FMT_EXPORT  between a block of FMT_BEGIN_EXPORT and FMT_END_EXPORT. Forgot to take it out.

Now that it I managed to get it working I will take it out, rebase and resolve conflicts as requested.

Reply:
Yeah, please remove this redundant `FMT_EXPORT` instead of commenting out.

Reply:
My apologies, I thought I had removed it in previous commit.